docs/docs/03-language-reference.md

```diff
@@ -388,6 +395,14 @@ Contact.fromJson(p);
 // - field "phone" is expected to be a string, got number.
 ```
 
+Same as with primitives, it is possible to opt-out of validation using `unsafe: true`:
+```js
+let p = Json { first: "Wing", phone: 1234 };
+let x = Contact.fromJson(p, unsafe: true);
+assert(x.last.len > 0);
+// RUNTIME ERROR: Cannot read properties of undefined (reading 'length')
```

Comment:
I think writing the exact error message in the language reference could lead to folks accidentally treating it as part of the public API. The message ("cannot read properties of undefined) is also confusing to anyone not familiar with JavaScript

```suggestion
assert(x.last.len > 0); // RUNTIME ERROR
```

Reply:
I agree, I was not thinking about it when copy pasta 😁